service/awsconfig/v5/resource/cloudformation/current.go

```diff
@@ -16,103 +18,129 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 		return StackState{}, microerror.Mask(err)
 	}
 
-	r.logger.LogCtx(ctx, "debug", "looking for AWS stack")
```

Comment:
This diff here is terrible. Maybe look at the whole file. It is not helpful because I put code into separate blocks for better readability. The separate blocks got their own comments at the head as well.

Reply:
https://github.com/giantswarm/aws-operator/pull/768/files?w=1